sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/ColumnResolutionHelper.scala

```diff
@@ -139,62 +139,64 @@ trait ColumnResolutionHelper extends Logging with DataTypeErrorsBase {
 
     val resolver = conf.resolver
 
-    def innerResolve(e: Expression, isTopLevel: Boolean): Expression = withOrigin(e.origin) {
```

Comment:
Most of the changes in this file are from re-indentation, so I recommend reviewing the whitespace-free diff instead: https://github.com/apache/spark/pull/46908/files?diff=unified&w=1